atomicapp/providers/openshift.py

```diff
@@ -371,13 +395,17 @@ def undeploy(self):
         logger.debug("Starting undeploy")
         delete_artifacts = []
         for kind, objects in self.openshift_artifacts.iteritems():
-            delete_artifacts.extend(objects)
+            # Add DCs to beggining of list so they are deleted first.
```

Comment:
s/beggining/beginning

Also can we add more explanation of why we do DCs first: "Do DC first because if you do RC before DC then the DC will re-spawn the RC before the DC is deleted."